website/static/js/comment.js

```diff
@@ -132,29 +132,61 @@ BaseComment.prototype.setupToolTips = function(elm) {
     });
 };
 
+BaseComment.prototype.fetchUserData = function() {
+    var self = this;
+    var request = osfHelpers.ajaxJSON(
+        'GET',
+        osfHelpers.apiV2Url('users/me/', {}),
+        {'isCors': true});
+    request.done(function(response) {
+        self.author = {
+            'id': response.data.id,
```

Comment:
We can probably eliminate this request by using the data in `contextVars`. You could pass in a user to the constructor of `BaseComment`.

```js
new BaseComment({user: {id: .., url: .., name: .., gravatarURL: ...}});
```